lib/python/src/bailo/helper/model.py

```diff
@@ -471,15 +471,24 @@ def from_mlflow(self, tracking_uri: str, experiment_id: str):
 
         logger.info(f"Successfully imported MLFlow experiment %s.", experiment_id)
 
-    def publish(self, mc_loc: str, run_id: str, semver: str = "0.1.0", notes: str = ""):
+    def publish(
+        self,
+        mc_loc: str,
+        semver: str = "0.1.0",
+        notes: str = "",
+        run_id: str | None = None,
+        select_by: str | None = None,
+    ):
         """Publishes a given experiments results to the model card.
 
         :param mc_loc: Location of metrics in the model card (e.g. performance.performanceMetrics)
-        :param run_id: Local experiment run ID to be selected
         :param semver: Semantic version of release to create (if artifacts present), defaults to 0.1.0 or next
         :param notes: Notes for release, defaults to ""
+        :param run_id: Local experiment run ID to be selected, defaults to None
+        :param select_by: String describing experiment to be selected (e.g. "MIN|MAX:accuracy"), defaults to None
```

Comment:
Format with this in accordance with MLFlow

Reply:
Done, now `accuracy MIN|MAX`